galaxy_ng/app/management/commands/set-repo-keyring.py

```diff
@@ -67,9 +68,15 @@ def handle(self, *args, **options):
                     self.echo("Process canceled.")
                     return
 
+        proc = subprocess.run([
+            "gpg", "--keyring", keyring_path, "--export", "-a"
+        ], capture_output=True)
+
+        pubkey = proc.stdout.decode().strip()
+
         task = dispatch(
-            set_repo_keyring,
-            kwargs={"repo_id": repo.pk, "keyring": keyring},
+            set_repo_gpgkey,
+            kwargs={"repo_id": repo.pk, "gpgkey": pubkey},
```

Comment:
Do I understand correctly that this is setting the field, renamed "gpgkey," to the pubkey captured from the subprocess command?

Reply:
Exactly. However i would call the field replaced, not renamed. It expects the key instead of a path to the key.